Api/src/routes/partyRoute.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
```

Comment:
You should call this file `partyRoutes`. It should be plural. since you have more than one route in there.